pkg/acsengine/defaults-kubelet.go

```diff
@@ -98,6 +98,13 @@ func setKubeletConfig(cs *api.ContainerService) {
 		}
 	}
 
+	// Get rid of values not supported in v1.12 and up
+	if common.IsKubernetesVersionGe(o.OrchestratorVersion, "1.12.0-alpha.1") {
```

Comment:
This should be `1.12.0-alpha.0` if we want to be rigorous (but otherwise doesn't matter)